src/main/scala/it/pagopa/interop/purposeprocess/api/impl/RiskAnalysisValidation.scala

```diff
@@ -44,8 +45,9 @@ object RiskAnalysisValidation {
     val sanitizedForm = form.copy(answers = form.answers.filter(_._2.nonEmpty))
 
     val validationRules: ValidationResult[List[ValidationEntry]] =
-      versions.lastOption
-        .fold[ValidationResult[List[ValidationEntry]]](TemplateVersionNotFound(tenantkind).invalidNec)(v =>
+      versions
+        .maxByOption(_._1)
```

Comment:
the map key is a String representing a number, ti should be converted before being sorted